test/kokoro/grpc_wallet.sh

```diff
@@ -11,7 +11,13 @@ pushd java
 ./gradlew build
 popd
 
-export GOPATH="${HOME}/gopath"
+# Download the latest Go version, extract it under /usr/local (which is the
+# default GOROOT) and copy `go` binary to /usr/bin which is part of PATH.
+wget https://dl.google.com/go/go1.16.5.linux-amd64.tar.gz
+tar -xvf go1.16.5.linux-amd64.tar.gz
+export GOROOT=${PWD}/go
+sudo cp ./go/bin/go /usr/bin/go
```

Comment:
Instead of potentially overriding existing /usr/bin/go, would it make sense to add this bin dir to the $PATH?

```suggestion
export PATH="${PWD}/go/bin:${PATH}"
```

Reply:
Done.